iocage_lib/ioc_start.py

```diff
@@ -661,6 +674,10 @@ def start_network_vnet_iface(self, nic, bridge, mtu, jid):
         :param jid: The jails ID
         :return: If an error occurs it returns the error. Otherwise, it's None
         """
+        vnet_default_interface = self.get('vnet_default_interface')
+        if vnet_default_interface == 'none':
+            vnet_default_interface = self.get_default_gateway()[1]
```

Comment:
That may cause a traceback if default route not configured

Reply:
This should be resolved now, I am raising an exception below if default route is not configured when we try to retrieve it